src/operations/extrusions/extrudeFromSlices.test.js

```diff
@@ -91,7 +91,7 @@ test('extrudeFromSlices (changing shape, changing dimensions)', t => {
   let geometry3 = extrudeFromSlices(
     {
       numberOfSlices: 5,
-      callback: (t, count) => {
+      callback: (percent, count, base) => {
```

Comment:
I have no opinion either way, but I am wondering: is putting the base/slice last the most logical ordering ?

Reply:
actually never mind, since that is always the same object for any number of slices , it is ok I guess